File: templates/scala-mongodb/model/src/main/scala/org/sgodden/tom/model/DeliveryDetails.scala
package org.sgodden.tom.model

import com.novus.salat.annotations.raw.Persist

case class DeliveryDetails() extends IDeliveryDetails {
  @Persist
  private var address: Address = null

  def getAddress = address

  def setAddress(address: IAddress) = this.address = address.asInstanceOf[Address]

}

File: templates/java-hibernate/model/src/main/java/org/sgodden/tom/model/DeliveryDetails.java
package org.sgodden.tom.model;


/**
 */
public class DeliveryDetails implements IDeliveryDetails {
    private Address address;

    public Address getAddress() {
        return address;
    }

    public void setAddress(IAddress address) {
        this.address = (Address) address;
    }
}
